container/support-scripts/build-all.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I do not think this file is needed and I rather keep the number of top level dirs small.

Reply:
This directory/file is only available within the `container/` directory for convenience and thus not stored as top-level directory.

Reply:
aah yes, my bad. This remark can be ignored.